playfab-docs/features/pricing/consumption-best-practices.md

```diff
@@ -44,9 +44,9 @@ We'll start with User Data and Statistics, since their usage patterns are simila
 
 ### Data and Statistics
 
-When evaluating your usage of User Data, the same approximation logic as for events applies–each 1 KB increases the meter count by 1–though keep in mind that each "element" of a call should be thought of as distinct for purposes of this calculation. Each key value pair in a call to update User Data, for example, is counted separately. For more specific details on the this, please refer to our [meter pricing documentation](meters/meters.md).
+When evaluating your usage of User Data, the same approximation logic as for events applies–each 1 KB increases the meter count by 1–though keep in mind that each "element" of a call should be thought of as distinct for purposes of this calculation. Each key value pair in a call to update User Data, for example, is counted separately. For reads, this 1 KB calculation applies to the total data returned, regardless of the number of key value pairs. So, writing 10 keys of 100 bytes each would be 10 "ticks" of the profile write meter, since each key value pair write is a minimum of 1 KB, while a read of those 10 keys would be 1 "tick", since it's a total of 1 KB. For more specific details on the this, please refer to our [meter pricing documentation](meters/meters.md).
```

Comment:
Each key value pair in a call to update User Data is counted separately. For reads, this 1 KB calculation applies to the total data returned, regardless of the number of key value pairs. This means that writing 10 keys of 100 bytes each is 10 "ticks" of the profile write meter, since each key value pair write is a minimum of 1 KB, while a read of those 10 keys is 1 "tick", since it's a total of 1 KB. For more specific details on the usages, see  [Pricing meters](meters/meters.md).